schema.json

```diff
@@ -30,37 +30,35 @@
                 ],
                 "additionalProperties": false
             }
-    },
-    "sets": {
-        "type": "array",
-        "minItems": 1,
-        "uniqueItems": true,
-        "items": {
+        },
+        "sets": {
             "type": "array",
             "minItems": 1,
-            "uniqeueItems": true,
+            "uniqueItems": true,
```

Comment:
hmm, I'm somewhat perplexed as to how this didn't trip a schema violation previously...nice catch